pages/components/button/button.css

```diff
@@ -1,19 +1,20 @@
 .sui_btn {
-  background: var(--PRIMARY-M3);
-  border: 2px solid var(--PRIMARY-M3);
+  background: var(--PRIMARY-M1);
+  border: 2px solid var(--PRIMARY-M1);
   border-radius: 5px;
   color: var(--WHITE);
-  padding: 1rem 1.5rem;
-  font-size: 1.5rem;
+  padding: 5px 15px;
+  font-size: 1.6rem;
   font-weight: 500;
   transition: all ease 0.5s;
 }
 .sui_btn:hover {
   cursor: pointer;
-  background: var(--PRIMARY-M2);
-  border: 2px solid var(--PRIMARY-M2);
+  /* background: var(--PRIMARY-M2);
+  border: 2px solid var(--PRIMARY-M2); */
```

Comment:
Remove commented code

Reply:
Thanks for pointing it out. It had to be uncommented with a different value. Will fix.

Reply:
Updated!